tracer/src/Datadog.Trace/Tags.cs

```diff
@@ -282,6 +296,11 @@ public static class Tags
         /// </summary>
         internal const string AmqpDeliveryMode = "amqp.delivery_mode";
 
+        /// <summary>
+        /// The bootstrap servers as defined in producer or consumer config
+        /// </summary>
+        internal const string KafkaBootstrapServers = "messaging.kafka.bootstrap.servers";
```

Comment:
So the good thing is that all our _existing_ tags have remappers so in the UI they all show up with a "messaging.kafka" prefix instead of "kafka":
- "kafka.group" => "meta.messaging.kafka.consumer_group"
- "kafka.tombstone" => "meta.messaging.kafka.tombstone"
- "kafka.offset" => "metrics.messaging.kafka.offset"
- "kafka.partition" => "metrics.messaging.kafka.partition"

However, the new tag "messaging.kafka.bootstrap.servers" has no such remapper, so we need to do that correctly now. We could consider changing the values of those tags now that there are remappers in place, but that could be an easy PR to do separately.

Reply:
oh yeah right. Completely forgot about the remapper. 
So no, don't touch the others (as the remapper may be partially broken, thus why I never merged the UNC PRs). And leave this new one as `messaging` indeed. Sorry for the noise